src/install/install.zig

```diff
@@ -558,10 +589,18 @@ const Task = struct {
         switch (this.tag) {
             .package_manifest => {
                 var allocator = bun.default_allocator;
+                const body = this.request.package_manifest.network.response_buffer.toOwnedSliceLeaky();
+                this.request.package_manifest.network.response_buffer.list.items.len = 0;
+                this.request.package_manifest.network.response_buffer.list.capacity = 0;
```

Comment:
Are these field initialisations missing before? 🤔

Reply:
I moved it into a method to clarify what this is doing

Basically, I want to just be really careful of any UAF caused later